CONTRIBUTING.md

```diff
@@ -36,4 +36,11 @@ Adding tools is easy once you understand the project structure. A few things nee
 4. Manually test your tool integration by running `agentstack tools add <your_tool>` and ensure it behaves as expected.
 
 ## Tests
```

Comment:
```markdown
## Tests
Tests are written in `tests` folder with the cli tests being in `tests/test_cli_loads.py`. 

Testing is sparse at the moment but contributions are very welcome! 🙏

You can run tests by first installing test dependencies with
```bash
pip install ".[test]"
```

Then running tox with
```bash
tox
```
```

Reply:
remove the escape characters on the ```s, the formatting got messed up on github :)

Reply:
Sure, working on this will solve this :)

Reply:
Done